vimeo-networking/src/main/java/com/vimeo/networking/Configuration.java

```diff
@@ -326,6 +339,36 @@ public Builder setLogLevel(LogLevel level) {
             return this;
         }
 
+        public Builder addNetworkInterceptor(Interceptor interceptor) {
+            if (interceptor != null) {
+                this.networkInterceptors = new ArrayList<>();
+                this.networkInterceptors.add(interceptor);
+            }
+            return this;
+        }
+
+        public Builder addNetworkInterceptors(List<Interceptor> interceptors) {
+            if (interceptors != null) {
+                this.networkInterceptors = new ArrayList<>(interceptors);
```

Comment:
Same with this. This may be good behavior is the method was named "setNetworkInterceptor", but as it stands, I would think the logic is that networkingInterceptors is already created, and this does a ```this.networkInterceptors.addAll(interceptors);```